test/testKit.spec.tsx

```diff
@@ -115,4 +117,17 @@ describe('App Host TestKit', () => {
         const host = createAppHostWithPacts([], [pactAPI])
         expect(host.getAPI(key).f()).toBe(1)
     })
+
+    describe('createAppHostAndWaitForLoading', () => {
+        it('should wait for loading all packages', async () => {
+            const host = await createAppHostAndWaitForLoading([dependsOnMockPackageEntryPoint, asyncLoadMockPackage], [])
+            expect(host.getAPI(MockPublicAPI)).toBeDefined()
+        })
+
+        it('should throw if failed to load all packages', async () => {
+            await expect(createAppHostAndWaitForLoading([dependsOnMockPackageEntryPoint], [])).rejects.toThrow(
+                new RegExp(MockPublicAPI.name)
+            )
+        })
+    })
```

Comment:
When testing timers you'd rather mock the clock, otherwise you'll make the build longer.
See - https://jestjs.io/docs/timer-mocks